lib/get-content-security-policy.js

```diff
@@ -0,0 +1,35 @@
+const { each, uniq, isArray } = require('lodash');
+
+const getContentSecurityPolicy = config => {
+  const csp = config.csp || [];
+  const directives = {
+    defaultSrc: [`'none'`],
+    styleSrc: [`'self'`],
+    imgSrc: [`'self'`],
+    fontSrc: [`'self'`, 'data:'],
+    scriptSrc: [`'self'`, (req, res) => `'nonce-${res.locals.static.nonce}'`]
+  };
+
+  const gaDirectives = {
+    scriptSrc: 'www.google-analytics.com',
+    imgSrc: 'www.google-analytics.com'
+  };
+
+  if (config.gaTagId) {
+    directives.scriptSrc = directives.scriptSrc.concat(gaDirectives.scriptSrc);
+    directives.imgSrc = directives.imgSrc.concat(gaDirectives.imgSrc);
+  }
```

Comment:
Can we kill all the GA stuff please?

Reply:
sure thang, I left it in as I figured we may add GA at some point, but can set in the client config anyhoo